en/editor/components/camera-component.md

```diff
@@ -36,3 +36,23 @@ The __Camera Component__ is an important functional component that we use to pre
 The __Camera__'s group rendering function works with the [Model Component](../../engine/renderable) through the __Visibility__ property of the [Camera Component](../../editor/components/camera-component.md). The user can set the __Visibility__ value through code to complete the group rendering. It should be noted that the __Visibility__ value is **bitwise comparison**, and the user can manipulate the **top 20 bits of Visibility** through **bit operations** to complete the grouping.
 
 The __Camera__ and models provided by default are all __rendered without grouping__. You do not need to change this value if the game has no special requirements to do so.
+
+### The Visibility property of the camera is set
+
+Set which tiers should be observed by this camera and which tiers should not be observed.
+
+Note: From version 3.0 of the Creator, the rendering of 2d elements (such as Sprite) also follows the Layer and Visibility judgement, so you can adjust the layer and visibility as required.
```

Comment:
Note:
-->
**Note**:

Reply:
From version 3.0 of the Creator, the
-->
The

I think we can remove: From version 3.0 of the Creator, because we are on v3 docs already.